src/Alert/__tests__/index.test.js

```diff
@@ -1,14 +1,13 @@
 // @flow
 import * as React from "react";
 import { mount } from "enzyme";
-import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
-import Alert from "../Alert";
+import Alert from "../index";
 
 describe("Alert", () => {
   it("should contain children", () => {
     const message = "Alert message";
-    const component = mount(<Alert theme={defaultTokens}>{message}</Alert>);
+    const component = mount(<Alert>{message}</Alert>);
```

Comment:
Can you rewrite this test to use `shallow` instead and fix that commented old stuff?

Reply:
Done